<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckTypeTailsDialog</class>
 <widget class="QDialog" name="CheckTypeTailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Change Airplane Tails</string>
  </property>
  <layout class="QGridLayout">
   <item row="4" column="4">
    <widget class="QPushButton" name="cancelButton_">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="effGroupLabel_">
     <property name="text">
      <string>Effectivity Group:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="helpButton_">
     <property name="text">
      <string>&amp;Help</string>
     </property>
     <property name="shortcut">
      <string>Alt+H</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="tailLabel_">
     <property name="text">
      <string>Choose airplane tails :</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="checkComboBox_"/>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="Horizontal Spacing2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>228</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="okButton_">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
     <property name="shortcut">
      <string>Alt+O</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="effGroupCheckBox_">
     <property name="text">
      <string>Select By Effectivity Group</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QComboBox" name="effGroupComboBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="checkLabel_">
     <property name="text">
      <string>Check Type :</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTreeWidget" name="airplaneTailsListView_">
     <property name="styleSheet">
      <string notr="true">QTreeWidget::branch{image:none;}</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>AirPlane Tail Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AirPlane Mfg Serial Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Airplane Customer Serial Number</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>helpButton_</tabstop>
  <tabstop>okButton_</tabstop>
  <tabstop>cancelButton_</tabstop>
  <tabstop>checkComboBox_</tabstop>
  <tabstop>effGroupCheckBox_</tabstop>
  <tabstop>effGroupComboBox_</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>helpButton_</sender>
   <signal>clicked()</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton_</sender>
   <signal>clicked()</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>effGroupCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>effGroupComboBox_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>effGroupCheckBox_</sender>
   <signal>toggled(bool)</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>selectByEffGroup(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>effGroupComboBox_</sender>
   <signal>activated(QString)</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>changeEffGroup(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkComboBox_</sender>
   <signal>activated(int)</signal>
   <receiver>CheckTypeTailsDialog</receiver>
   <slot>enableOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
